src/engine/srv.c

```diff
@@ -491,6 +492,15 @@ dss_srv_handler(void *arg)
 		if (dss_xstream_exiting(dx))
 			break;
 
+		if (dx->dx_xs_id == 0) {
+			uint64_t now = sched_cur_msec();
+
+			/* dump memory consumption every minute */
+			if (now > then + 60 * 1000) {
+				then = now;
+				dm_mem_dump_log();
+			}
```

Comment:
debugging code?

Reply:
it's an empty function in release mode, but enabled in CI

Reply:
ah, hmm, probably only dump if it reach to some threshold.